examples/nimble-typescript/start/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Unless someone has a reason not to, I think we should update the root `package.json` to build and lint the `examples` directory. Building will help us ensure the example stays up to date if we make breaking changes.

It would also be good to enable linting to help catch errors and keep the code readable.

Reply:
I'm working on this in a branch